Watch Extension/MRSessionProgressGroupRenderer.swift

```diff
@@ -11,12 +11,18 @@ class MRSessionProgressGroupRenderer : NSObject {
         self.timer = NSTimer.scheduledTimerWithTimeInterval(1, target: self, selector: "update", userInfo: nil, repeats: true)
     }
     
-    deinit {
+    func deactivate() {
         self.timer?.invalidate()
         self.timer = nil
     }
     
+    deinit {
+        deactivate()
+    }
+    
     func update() {
+        MRExtensionDelegate.sharedDelegate().applicationDidBecomeActive()
```

Comment:
Is it necessary to call that here , it forces the sending of the data every time the scheduler call the update method ?